components/calendar/test/calendar.visual-diff.js

```diff
@@ -19,12 +19,12 @@ describe('d2l-calendar', () => {
 
 	after(() => browser.close());
 
-	it('has no selected-value when selected-value not set', async function() {
+	it('no selected value', async function() {
```

Comment:
Most changes in here were to make test names smaller and make this generally more readable